[BUG] : Error: Cannot find module 'cli-ux'

**Describe the bug**
When a GitHub action command is performed the CLI returns `Error: Cannot find module 'cli-ux'`, This package was not required to install during the development phase. 

**To Reproduce**
Steps to reproduce the behavior:
1. Execute the command `daily-cli github configure`
2. Type your github token
2. Execute the command `daily-cli github list`

**Expected behavior**
Show the repositories ligated to my user. 

**Screenshots**
![image](https://user-images.githubusercontent.com/6563162/96502691-c692f680-1217-11eb-9696-3e1188073337.png)


**Desktop (please complete the following information):**
 - OS: MacOs Catalina
 - CLI Version 0.0.6

--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ $ npm install -g daily-cli
 $ daily-cli COMMAND
 running command...
 $ daily-cli (-v|--version|version)
-daily-cli/0.0.6 darwin-x64 node-v12.16.2
+daily-cli/0.0.7 darwin-x64 node-v12.16.2
 $ daily-cli --help [COMMAND]
 USAGE
   $ daily-cli COMMAND
@@ -105,10 +105,12 @@ DESCRIPTION
   Extra documentation goes here
 ```
 
-_See code: [src/commands/api-generator.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/api-generator.js)_
+_See code: [src/commands/api-generator.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/api-generator.js)_
 
 ## `daily-cli aws-accounts ACTION`
-![](https://s3.amazonaws.com/public.flavioaandres.com/aws-accounts_to.gif) 
+
+Describe the command here
+
 ```
 USAGE
   $ daily-cli aws-accounts ACTION
@@ -126,10 +128,7 @@ DESCRIPTION
   Allow administrating your AWS accounts
 ```
 
-## See all profiles loaded: 
-![](https://s3.us-east-1.amazonaws.com/public.flavioaandres.com/aws-accounts_show.gif)
-
-_See code: [src/commands/aws-accounts.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/aws-accounts.js)_
+_See code: [src/commands/aws-accounts.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/aws-accounts.js)_
 
 ## `daily-cli configure`
 
@@ -150,9 +149,11 @@ DESCRIPTION
   Extra documentation goes here https://github.com/FlavioAandres/daily-cli
 ```
 
-_See code: [src/commands/configure.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/configure.js)_
+_See code: [src/commands/configure.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/configure.js)_
 
 ## `daily-cli github ACTION`
+
+...
 
 ```
 USAGE
@@ -184,11 +185,11 @@ DESCRIPTION
      Required access: repo & delete_repo
 ```
 
-_See code: [src/commands/github.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/github.js)_
+_See code: [src/commands/github.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/github.js)_
 
 ## `daily-cli health`
 
-![](https://s3.amazonaws.com/public.flavioaandres.com/health_1.gif)
+Describe the command here
 
 ```
 USAGE
@@ -216,9 +217,11 @@ EXAMPLES
   $ health -s github trello
 ```
 
-_See code: [src/commands/health.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/health.js)_
+_See code: [src/commands/health.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/health.js)_
 
 ## `daily-cli hello`
+
+Describe the command here
 
 ```
 USAGE
@@ -235,7 +238,7 @@ EXAMPLE
   $ daily-cli hello --name pecue
 ```
 
-_See code: [src/commands/hello.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/hello.js)_
+_See code: [src/commands/hello.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/hello.js)_
 
 ## `daily-cli help [COMMAND]`
 
@@ -270,5 +273,5 @@ DESCRIPTION
   Extra documentation goes here
 ```
 
-_See code: [src/commands/run.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.6/src/commands/run.js)_
+_See code: [src/commands/run.js](https://github.com/FlavioAandres/daily-cli/blob/v0.0.7/src/commands/run.js)_
 <!-- commandsstop -->

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "daily-cli",
   "description": "cli to make the developer life easier",
-  "version": "0.0.6",
+  "version": "0.0.7",
   "author": "Flavio A @FlavioAandres",
   "bin": {
     "daily-cli": "./bin/run"
@@ -15,6 +15,7 @@
     "boxen": "^4.2.0",
     "chalk": "^4.1.0",
     "cli-table3": "^0.6.0",
+    "cli-ux": "^5.5.0",
     "ejs": "^3.1.5",
     "enquirer": "^2.3.6",
     "execa": "^4.0.3",

